### ISSUE TITLE
4장 softmax_cross_entropy_with_logits_v2 질문입니다.

### ISSUE THREAD

**Issue by rycont** [golbin/TensorFlow-Tutorials#54]
python 3.6.8에 tf 1.2.0입니다. `cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits_v2(labels=Y, logits=model))`라는 코드(70페이지 2번째 줄 혹은 [깃헙링크](https://github.com/golbin/TensorFlow-Tutorials/blob/master/04%20-%20Neural%20Network%20Basic/02%20-%20Deep%20NN.py#L47))에서  ` module 'tensorflow.python.ops.nn' has no attribute 'softmax_cross_entropy_with_logits_v2'`라는 오류가 납니다. 왜 저런 오류가 나는지 궁금합니다.

**Comment by dhnam** [comment]
https://github.com/tensorflow/docs/tree/r1.2/site/en/api_docs/python/tf/nn

해당 함수는 tf1.2.0에는 없었습니다. `softmax_cross_entropy_with_logits`로 바꾸던가, 해당 함수가 지원되는 tf1.5 이상의 버전으로 업데이트해야 합니다.
